zabbix-mysql-dump

```diff
@@ -189,33 +220,50 @@ while read table; do
 	else
 		dump_opt="--extended-insert=FALSE"
 	fi
-	mysqldump --routines --opt --single-transaction --skip-lock-tables \
-		$dump_opt $MYSQL_CONN --tables ${table} >>"${DUMPFILE}"
-
-	# show percentage
-	i=$((i+1)); i_percent=$(($i * 100 / $DB_TABLE_NUM))
-	if [ $(($i_percent % 12)) -eq 0 ]; then
-		echo -n "${i_percent}%"
-	else
-		if [ $(($i_percent % 2)) -eq 0 ]; then echo -n "."; fi
+	TABLE_DUMP_CMD="mysqldump ${MYSQL_CONN} --routines --opt --single-transaction --skip-lock-tables $dump_opt $DBNAME --tables ${table}"
```

Comment:
I'm getting this error when not using the conf file.

```
# ./zabbix-mysql-dump -n -h 127.0.0.1 -u zabbix -pzabbix -d zabbix -o /root/
Configuration:
 - host:     127.0.0.1 (127.0.0.1)
 - database: zabbix
 - user:     zabbix
Fetching list of existing tables...
Starting table backups...
mysqldump: Couldn't find table: "zabbix"
Failed dumping table acknowledges. Aborting.
````

I found that $DBNAME is already on ${MYSQL_CONN}

The TABLE_DUMP_CMD is like this `mysqldump -h 127.0.0.1 -u zabbix -pzabbix zabbix --routines --opt --single-transaction --skip-lock-tables --no-data zabbix --tables acknowledges`

Where `zabbix` appears two times, and the command tries to find a table named `zabbix`, and ends with an error.